src/components/Header/HeaderRight/HeaderRightOption/HeaderRightOption.js

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import './HeaderRightOption.css';
+
+ class HeaderRightOption extends Component {
+  
+  render() {
+    // console.log(this.props);
+    return <div className='tab_option'>
+        <span className='tab_name'> {this.props.details.name}</span>
```

Comment:
Please go and read defaultProps and propTypes in React and add that here.

Reply:
HeaderRightOption.defaultProps = {
 details: {}
}

HeaderRightOption.propTypes = {
 details: PropTypes.object,
}